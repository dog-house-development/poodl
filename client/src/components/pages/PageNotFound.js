import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class PageNotFound extends Component {
    render() {
        return (
            <div className="landing-container">
                <div className="landing-header">
                    <h1 className="landing-title">404</h1>
                    <h1>Page not found</h1>
                    <p>
                        If you believe this occured in error, please contact support at{' '}
                        <a href="mailto:doghousedevelop@gmail.com" target="_blank" rel="noopener">
                            doghousedevelop@gmail.com.
                        </a>
                    </p>
                    <h3>Thanks</h3>
                    <Link to="/dashboard" className="button large primary">
                        Home
                    </Link>
                </div>
            </div>
        );
    }
}

export default PageNotFound;
