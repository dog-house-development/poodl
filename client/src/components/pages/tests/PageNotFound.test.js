import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import _ from 'lodash';

import PageNotFound from '../PageNotFound';

configure({ adapter: new Adapter() });

describe('PageNotFound tests', () => {
    let wrapper, instance;
    const setInstanceAndWrapper = () => {
        wrapper = shallow(<PageNotFound />);
        instance = wrapper.instance();
    };
    beforeEach(() => {
        setInstanceAndWrapper();
    });

    describe('render', () => {
        it('should render correctly', () => {
            expect(wrapper).toMatchSnapshot();
        });
    });
});
