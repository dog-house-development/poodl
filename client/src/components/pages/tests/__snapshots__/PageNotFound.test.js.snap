// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound tests render should render correctly 1`] = `
<div
  className="landing-container"
>
  <div
    className="landing-header"
  >
    <h1
      className="landing-title"
    >
      404
    </h1>
    <h1>
      Page not found
    </h1>
    <p
      className="landing-info"
    >
      If you believe this occured in error, please contact support at
       
      <a
        href="mailto:doghousedevelop@gmail.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        doghousedevelop@gmail.com.
      </a>
    </p>
    <h3
      className="landing-info"
    >
      Thanks
    </h3>
    <Connect(GoHome)
      buttonOrLink="button"
      size="large"
    />
  </div>
</div>
`;
